tests/hpc/mpi_imb.pm

```diff
@@ -0,0 +1,114 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Basic MPI integration test using IMB.
+# Maintainer: Kernel QE <kernel-qa@suse.de>
+
+use Mojo::Base qw(hpcbase hpc::utils), -signatures;
+use testapi;
+use serial_terminal qw(select_serial_terminal select_user_serial_terminal);
+use lockapi;
+use utils;
+use registration;
+use version_utils 'is_sle';
+use Utils::Logging 'export_logs';
+use hpc::formatter;
+use isotovideo;
+
+use POSIX 'strftime';
+
+sub run ($self) {
+    select_serial_terminal();
+    my $mpi = get_required_var('MPI');
+    my $mpi2load = '';
+    my %exports_path = (
+        bin => '/home/bernhard/bin',
+        hpc_lib => '/usr/lib/hpc',
+    );
+    my $user_virtio_fixed = isotovideo::get_version() >= 35;
+    my $prompt = $user_virtio_fixed ? $testapi::username . '@' . get_required_var('HOSTNAME') . ':~> ' : undef;
+
+    script_run("sudo -u $testapi::username mkdir -p $exports_path{bin}");
+    zypper_call("in imb-gnu-$mpi-hpc");
+
+    type_string('pkill -u root', lf => 1) unless $user_virtio_fixed;
+    select_user_serial_terminal($prompt);
```

Comment:
Why we have to do `$user_virtio_fixed`, is it some kind of workaround and where is it coming from?